test/e2e/diff_test.go

```diff
@@ -0,0 +1,65 @@
+package integration
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman rmi", func() {
```

Comment:
The file name is diff_test but this looks to be all rmi tests?

Reply:
fixed